app/views/course_modules/edit.html.erb

```diff
@@ -1,6 +1,9 @@
 <%= render "shared/components/desktop_layout" do %>
-  <div class="box">
-    <%= link_to "Back", course_module_path(@course, @course_module), class: "nav-link" %>
+  <div class="flex gap-4 items-center ">
+    <%= link_to course_module_path(@course, @course_module),  class: "nav-link" do %>
+    <span class="icon icon-arrow-left-gray icon-small"></span>
+    <% end %>
+    <div class="font-medium text-sm">Back</div>
```

Comment:
this should be a span and part of the block above, does the navigation work on clicking the text Back ?

Reply:
Created back button component and it is using here